### ISSUE TITLE
gateway.networking.k8s.io/channel: experimental causing ArgoCD to be out of sync on GCP

### ISSUE THREAD

**Issue by pupca** [traefik/traefik-helm-chart#1158]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

30.0.2

### What version of Traefik are you using?

3.1.0

### What did you do?

I am deploying Traefik to GCP via ArgoCD that is putting my application "out of sync". The reason is that Traefik is trying to use the `gateway.networking.k8s.io/channel: experimental` that is not supported by GCP. only `gateway.networking.k8s.io/channel: standard` is supported.

<img width="1507" alt="Screenshot 2024-08-05 at 10 55 33" src="https://github.com/user-attachments/assets/0e6a8e8f-8b36-466c-9bad-1367bbf35e97">

### What did you see instead?

The application in ArgoCD is out of sync. I expect it to by "Synced"

### What is your environment & configuration?

values.yaml:
```values.yaml
ports:
  websecure:
    appProtocol: HTTPS # Hint for Google L7 load balancer
service:
  type: ClusterIP
extraObjects:
  - apiVersion: gateway.networking.k8s.io/v1beta1
    kind: Gateway
    metadata:
      name: traefik
      annotations:
        networking.gke.io/certmap: "wildcard-certificate-map"

    spec:
      gatewayClassName: gke-l7-global-external-managed
      addresses:
        - type: NamedAddress
          value: "ip-address"
      listeners:
        - name: https
          protocol: HTTPS
          port: 443
          allowedRoutes:
            namespaces:
              from: All
  - apiVersion: gateway.networking.k8s.io/v1beta1
    kind: HTTPRoute
    metadata:
      name: traefik
    spec:
      parentRefs:
        - kind: Gateway
          name: traefik
      rules:
        - backendRefs:
            - name: traefik
              port: 443
  - apiVersion: networking.gke.io/v1
    kind: HealthCheckPolicy
    metadata:
      name: traefik
    spec:
      default:
        config:
          type: HTTP
          httpHealthCheck:
            port: 9000
            requestPath: /ping
      targetRef:
        group: ""
        kind: Service
        name: traefik

```

**Comment by mloiseleur** [comment]
Thanks for this detailed bugreport.
Since Traefik is now passing all conformance tests on v1.1.0, we are thinking to switch standard channel v1.1.0.
Users wanting to use _TCPRoute_ or _TLSRoute_ will need to install experimental CRD and [enable the experimental channel](https://doc.traefik.io/traefik/providers/kubernetes-gateway/#experimentalchannel) in the provider.
Would this solve your issue ?

**Comment by pupca** [comment]
@mloiseleur yes, that would be expected behaviour that if you want to have experimental features you need to enable the channel.

Thanks.
